src/pages/index.js

```diff
@@ -32,53 +155,43 @@ const addModalFormValidator = new FormValidator(
   variables.addModalForm
 );
 addModalFormValidator.enableValidation();
-//------------------------------------------------------------------------------
-const editFormPopup = new PopupWithForm("#profile-edit-modal", (formData) => {
-  user.setUserInfo({
-    name: formData.title,
-    job: formData.description,
-  });
-});
-
-editFormPopup.setEventListeners();
 
-const user = new UserInfo({
-  nameSelector: "#profile-info-title",
-  jobSelector: "#profile-info-description",
-});
-
-variables.profileEditButton.addEventListener("click", () => {
-  const userInfo = user.getUserInfo();
-  variables.profileModalNameInput.value = userInfo.name.trim();
-  variables.profileModalDescriptionInput.value = userInfo.job.trim();
-  editFormPopup.open();
-});
+const avatarFormValidator = new FormValidator(
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form (if you don’t have it then you need to add it into each form in `index.html` first)
    const formName = formElement.getAttribute('name')

   // here you store the validator using the `name` of the form
    formValidators[formName] = validator;
    validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```